controllers/cinder_controller.go

```diff
@@ -67,9 +67,9 @@ func (r *CinderReconciler) GetKClient() kubernetes.Interface {
 	return r.Kclient
 }
 
-// GetLogger -
-func (r *CinderReconciler) GetLogger() logr.Logger {
-	return r.Log
+// getLog returns a logger object with a logging prefix of "conrollers.name" and aditional controller context fields
+func (r *CinderReconciler) getLog(ctx context.Context) logr.Logger {
```

Comment:
Every controller has a set of GetXXX() functions at the top, which have a unique signature based on their reconciler structure. I think we should retain that pattern, and not rename each controller's GetLogger() function to getLog().

In other words, can we keep the GetLogger() name?